nemo/collections/asr/parts/submodules/token_classifier.py

```diff
@@ -81,8 +88,12 @@ def __init__(
         )
         self.post_init(use_transformer_init=use_transformer_init)
 
+    def set_log_softmax_enabled(self, value: bool) -> "TokenClassifier":
```

Comment:
Add the setter and getter decorators for this. Also `self._log_softmax`. We break enough properties that should use the underscoring convention.

Reply:
Ooh, good call.